app/src/main/java/com/imobile3/groovypayments/ui/main/MainDashboardActivity.java

```diff
@@ -102,7 +103,8 @@ private void handleDashboardButtonClick(@NonNull MainDashboardButton button) {
                 break;
 
             case Placeholder2:
-                throw new RuntimeException("User clicked a Placeholder button");
+                PlaceholderDialogFragment placeholderDialogFragment = new PlaceholderDialogFragment();
```

Comment:
Also provide a screenshot of the dialog in the PR description once completed

Reply:
![Screenshot_20210724-230751_Groovy Mobile Payments](https://user-images.githubusercontent.com/69394115/126872808-c4b063be-caa2-42e2-b6ad-cb1bd70c5f60.jpg)